instrumented/Components/Loading/Loading.js

```diff
@@ -0,0 +1,13 @@
+import { Fragment } from 'react';
+import './Loading.css';
+import loading from '../../Assets/loading.png';
+
+const Loading = () => {
+  return (
+    <div className="loading-icon-container">
+      <img className="loading-icon" src={loading} alt="loading icon" />
+    </div>
+  )
+}
+
+export default Loading;
```

Comment:
I didn't see this file being used anywhere in the tests. Can you please explain what this file is for and if it's needed?

Reply:
'Explained' [here](https://github.com/mi-voz-mi-voto/mi-voz-mi-voto/pull/30#discussion_r804866273)